components/ModEditSubmissionTopbar.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div class="flex flex-row justify-between w-full">
+        <div>
+            <button @click="copyId"
+                class="flex flex-row w-90 bg-neutral p-2 m-2 border-2 border-slate-400 rounded hover">
+                ID: {{ id }}
+                <SuccessCheckMark v-if="copySuccess" role="img" title="clipboard copy" class="ml-2 w-6" />
+                <CopyContent v-else role="img" title="successful copy" class="ml-2 w-6" />
+            </button>
+        </div>
+        <div class="flex flex-row justify p-2 font-bold ">
+            <button
+                class="flex justify-center items-center rounded-full bg-white border-slate-400 border-2 w-28 text-sm mr-2 ">Save
+                & Exit</button>
```

Comment:
I would add this string to the locale files. The "&" looks like it could potentially cause problems. Git doesn't seem too happy with it. 

<img width="256" alt="Screenshot 2024-05-28 at 12 45 09 AM" src="https://github.com/ourjapanlife/findadoc-web/assets/31802656/bf88e180-4d99-4ee2-88ca-a65bfd1e62ad">

Reply:
Moved all the text except ID to the text.json file. No Japanese but has the translation string for now